packages/amplify-util-uibuilder/commands/cloneComponentsFromEnv.js

```diff
@@ -15,63 +15,60 @@ async function run(context) {
   }
   const existingComponentsNewEnv = await listUiBuilderComponents(context, newEnvName);
   if (existingComponentsNewEnv.entities.length > 0) {
-    logger.info(`${existingComponentsNewEnv.entities.length} components already exist in new env. Skipping creation of local components.`);
+    printer.debug(
+      `${existingComponentsNewEnv.entities.length} components already exist in new env. Skipping creation of local components.`,
+    );
     return;
   }
 
-  try {
-    const environmentName = args.environmentName ? args.environmentName : context.exeInfo.localEnvInfo.envName;
-
-    const appId = args.appId ? args.appId : context.exeInfo.teamProviderInfo[environmentName].awscloudformation.AmplifyAppId;
+  const environmentName = args.environmentName ? args.environmentName : context.exeInfo.localEnvInfo.envName;
```

Comment:
nit: this and the next line can be
```suggestion
  const environmentName = args.environmentName || context.exeInfo.localEnvInfo.envName;
```